app/src/main/java/com/courseproject/tindar/entities/Message.java

```diff
@@ -0,0 +1,63 @@
+package com.courseproject.tindar.entities;
+
+import java.sql.Timestamp;
+
+public class Message {
+    private String messageID;
+    private String text;
+    private Timestamp timestamp;
+    private String sentFrom;
+    private String sentTo;
+    private String conversationId;
+
+    // Getter methods
+    public String getMessageID() {
+        return messageID;
+    }
+
+    public String getText() {
+        return text;
+    }
+
+    public Timestamp getTimestamp() {
+        return timestamp;
+    }
+
+    public String getSentFrom() {
+        return sentFrom;
+    }
+
+    public String getSentTo() {
+        return sentTo;
+    }
+
+    public String getConversationId() {
+        return conversationId;
+    }
+
+    // Setter methods
+    public void setMessageID(String messageID) {
```

Comment:
Can we use setMessageId instead of setMessageID? and messageId instead of messageID

Reply:
... ngl that's weird to me and I'd prefer it the way it is